src/rust/runtime/queue/mod.rs

```diff
@@ -44,7 +44,7 @@ impl IoQueueTable {
     /// When Demikernel is interposing system calls of the underlying OS
     /// this offset enables us to distinguish I/O queue descriptors from
     /// file descriptors.
```

Comment:
Is this really necessary for something?  (I know this is existing code, but we've shrunk the number space from 2^64 to 2^32.  And now this limits the queue descriptor space even more).

Reply:
Yes it is and this will be clear when the SHIM layer comes into play.

Reply:
Got it, thanks.  Wanted to make sure this wasn't just a debugging aid.